web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-hero.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-lzpvnd.php, but found phpcs-scan-lZpvnD.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).